src/perf.rs

```diff
@@ -0,0 +1,175 @@
+    //
+    // Id is one of:
+    //
+    // enum {
+    // 	rocksdb_user_key_comparison_count = 0,
+    // 	rocksdb_block_cache_hit_count,
+    // 	rocksdb_block_read_count,
+    // 	rocksdb_block_read_byte,
+    // 	rocksdb_block_read_time,
+    // 	rocksdb_block_checksum_time,
+    // 	rocksdb_block_decompress_time,
+    // 	rocksdb_get_read_bytes,
+    // 	rocksdb_multiget_read_bytes,
+    // 	rocksdb_iter_read_bytes,
+    // 	rocksdb_internal_key_skipped_count,
+    // 	rocksdb_internal_delete_skipped_count,
+    // 	rocksdb_internal_recent_skipped_count,
+    // 	rocksdb_internal_merge_count,
+    // 	rocksdb_get_snapshot_time,
+    // 	rocksdb_get_from_memtable_time,
+    // 	rocksdb_get_from_memtable_count,
+    // 	rocksdb_get_post_process_time,
+    // 	rocksdb_get_from_output_files_time,
+    // 	rocksdb_seek_on_memtable_time,
+    // 	rocksdb_seek_on_memtable_count,
+    // 	rocksdb_next_on_memtable_count,
+    // 	rocksdb_prev_on_memtable_count,
+    // 	rocksdb_seek_child_seek_time,
+    // 	rocksdb_seek_child_seek_count,
+    // 	rocksdb_seek_min_heap_time,
+    // 	rocksdb_seek_max_heap_time,
+    // 	rocksdb_seek_internal_seek_time,
+    // 	rocksdb_find_next_user_entry_time,
+    // 	rocksdb_write_wal_time,
+    // 	rocksdb_write_memtable_time,
+    // 	rocksdb_write_delay_time,
+    // 	rocksdb_write_pre_and_post_process_time,
+    // 	rocksdb_db_mutex_lock_nanos,
+    // 	rocksdb_db_condition_wait_nanos,
+    // 	rocksdb_merge_operator_time_nanos,
+    // 	rocksdb_read_index_block_nanos,
+    // 	rocksdb_read_filter_block_nanos,
+    // 	rocksdb_new_table_block_iter_nanos,
+    // 	rocksdb_new_table_iterator_nanos,
+    // 	rocksdb_block_seek_nanos,
+    // 	rocksdb_find_table_nanos,
+    // 	rocksdb_bloom_memtable_hit_count,
+    // 	rocksdb_bloom_memtable_miss_count,
+    // 	rocksdb_bloom_sst_hit_count,
+    // 	rocksdb_bloom_sst_miss_count,
+    // 	rocksdb_key_lock_wait_time,
+    // 	rocksdb_key_lock_wait_count,
+    // 	rocksdb_env_new_sequential_file_nanos,
+    // 	rocksdb_env_new_random_access_file_nanos,
+    // 	rocksdb_env_new_writable_file_nanos,
+    // 	rocksdb_env_reuse_writable_file_nanos,
+    // 	rocksdb_env_new_random_rw_file_nanos,
+    // 	rocksdb_env_new_directory_nanos,
+    // 	rocksdb_env_file_exists_nanos,
+    // 	rocksdb_env_get_children_nanos,
+    // 	rocksdb_env_get_children_file_attributes_nanos,
+    // 	rocksdb_env_delete_file_nanos,
+    // 	rocksdb_env_create_dir_nanos,
+    // 	rocksdb_env_create_dir_if_missing_nanos,
+    // 	rocksdb_env_delete_dir_nanos,
+    // 	rocksdb_env_get_file_size_nanos,
+    // 	rocksdb_env_get_file_modification_time_nanos,
+    // 	rocksdb_env_rename_file_nanos,
+    // 	rocksdb_env_link_file_nanos,
+    // 	rocksdb_env_lock_file_nanos,
+    // 	rocksdb_env_unlock_file_nanos,
+    // 	rocksdb_env_new_logger_nanos,
+    // 	rocksdb_total_metric_count = 68
+    // }
+    pub fn metric(&self, id: c_int) -> u64 {
```

Comment:
There are a lot of variants here, but I would still prefer to use an enum (with the `#[non_exhaustive]` attribute).